doc/manual/src/installation/uninstall.md

```diff
@@ -1,12 +1,52 @@
 # Uninstalling Nix
 
+## Sections
+- [Single User](#single-user)
+  - [Manual Method](#manual-method)
+    - [Step 1: Restore Modified Files (if applicable)](#step-2-restore-modified-files-if-applicable)
+    - [Step 2: Delete Nix Files](#step-4-delete-nix-files)
+- [Multi User](#multi-user)
+    - [Linux](#linux)
+    - [macOS](#macos)
+
+
 ## Single User
+### If you have a [single-user installation](./installing-binary.md#single-user-installation) of Nix
+### Manual Method
 
-If you have a [single-user installation](./installing-binary.md#single-user-installation) of Nix, uninstall it by running:
+#### Step 1: Restore Modified Files (if applicable)
+
+1. Nix may have modified certain files on your system. If you have created backup copies of these files before installing Nix, you can restore them. Here are the instructions for bash and zsh files:
+
+   - Bash: Nix may have modified the `/etc/bash.bashrc` or `~/.bash_profile` file. If you have created a backup of this file before installing Nix, you can restore it using the following commands:
+
+     ```bash
+     sudo cp /etc/bash.bashrc.backup-before-nix /etc/bash.bashrc
+     sudo cp ~/.bash_profile.backup-before-nix ~/.bash_profile
```

Comment:
Why not

```suggestion
     sudo mv /etc/bash.bashrc.backup-before-nix /etc/bash.bashrc
     sudo mv ~/.bash_profile.backup-before-nix ~/.bash_profile
```

Because if those files exist we're done; otherwise error showing they don't